src/scraper/spiders/cpp_symbol_spider.py

```diff
@@ -17,6 +17,28 @@
 
 RETURN_VALUE_HEADER = b'<span class="mw-headline" id="Return_value">Return value</span>'
 
+def clean(string, *remove):
+    clean_str = unescape(remove_tags(string))
+    for item in remove:
+        clean_str = clean_str.replace(item, "")
```

Comment:
.. didn't we talk about using something like ''.join(c for c in clean_str if c not in remove) earlier?